moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -167,48 +184,71 @@ class TrajectoryExecutionManager
 
   /// Add a trajectory for immediate execution. Optionally specify a controller to use for the trajectory. If no
   /// controller is specified, a default is used. This call is non-blocking.
-  bool pushAndExecute(const moveit_msgs::RobotTrajectory& trajectory, const std::string& controller = "");
+  /// Optionally specify an expiration time for the trajectory, from reception to execution (default 60 secs).
+  /// Optionally specify callback that is called when the execution of the trajectory is completed.
+  bool pushAndExecuteSimultaneous(const moveit_msgs::RobotTrajectory& trajectory, const std::string& controller = "",
```

Comment:
```suggestion
  bool pushAndExecuteSimultaneously(const moveit_msgs::RobotTrajectory& trajectory, const std::string& controller = "",
```
any native speaker around to comment?